Add ru_ru.json

VGamerGroup opened the pull request:
I translated this mod into Russian Language. Please, will add this traslation.

VGamerGroup committed:
Create ru_ru.json

I translated this mod into Russian Language. Please, will add this traslation.

ghost commented:
`"createdieselgenerators.ponder.large_diesel_engine.text_4": "Они будут создавать нагрузку пропорционально тому, сколько двигателей вы установите."`
Вернее будет сказать так:
`"createdieselgenerators.ponder.large_diesel_engine.text_4": "Допустимая нагрузка будет увеличиваться пропорционально количеству установленных двигателей"`

VGamerGroup commented:
> `"createdieselgenerators.ponder.large_diesel_engine.text_4": "Они будут создавать нагрузку пропорционально тому, сколько двигателей вы установите."` Вернее будет сказать так: `"createdieselgenerators.ponder.large_diesel_engine.text_4": "Допустимая нагрузка будет увеличиваться пропорционально количеству установленных двигателей"`

Спасибо за поправку! А но получалось, что двигатели - это потребители силы вращения, а не её производители. Отмечу вас в файле перевода. 
🤝 Взаимопомощь

VGamerGroup committed:
Update ru_ru.json

Fix 1:
Text of line createdieselgenerators.ponder.large_diesel_engine.text_4 was changed from (Они будут создавать нагрузку пропорционально тому, сколько двигателей вы установите.) to (Допустимая нагрузка будет увеличиваться пропорционально количеству установленных двигателей) by player ProTerUga on 22.08.2023"
}

VGamerGroup committed:
Update ru_ru.json

comma was not included

VGamerGroup commented:
Thank you!
